tests_metricflow/snapshots/test_predicate_pushdown_rendering.py/SqlQueryPlan/DuckDB/test_simple_join_to_time_spine_pushdown_filter_application__plan0_optimized.sql

```diff
@@ -0,0 +1,39 @@
+-- Constrain Output with WHERE
+-- Compute Metrics via Expressions
+SELECT
+  metric_time__day
+  , booking__is_instant
+  , bookings AS bookings_join_to_time_spine
+FROM (
+  -- Join to Time Spine Dataset
+  SELECT
+    subq_15.ds AS metric_time__day
+    , subq_13.booking__is_instant AS booking__is_instant
+    , subq_13.bookings AS bookings
+  FROM ***************************.mf_time_spine subq_15
+  LEFT OUTER JOIN (
+    -- Constrain Output with WHERE
+    -- Pass Only Elements: ['bookings', 'booking__is_instant', 'metric_time__day']
+    -- Aggregate Measures
+    SELECT
+      metric_time__day
+      , booking__is_instant
+      , SUM(bookings) AS bookings
+    FROM (
+      -- Read Elements From Semantic Model 'bookings_source'
+      -- Metric Time Dimension 'ds'
+      SELECT
+        DATE_TRUNC('day', ds) AS metric_time__day
+        , is_instant AS booking__is_instant
+        , 1 AS bookings
+      FROM ***************************.fct_bookings bookings_source_src_28000
+    ) subq_10
+    WHERE booking__is_instant
+    GROUP BY
+      metric_time__day
+      , booking__is_instant
+  ) subq_13
+  ON
+    subq_15.ds = subq_13.metric_time__day
+) subq_16
+WHERE booking__is_instant
```

Comment:
Reminding myself why we need this second where filter. I think it's because the time spine join might add null rows where `booking__is_instant = null`.

Reply:
Yes, that's correct.